lbz/resource.py

```diff
@@ -134,13 +144,14 @@ def __call__(self) -> Response:
             resp.headers.update(self.resp_headers())
         return resp
 
-    def _get_allowed_origins(self, origins: list) -> str:
+    def _get_allowed_origins(self, origins: Union[List[str], str]) -> str:
```

Comment:
Are you sure it can be a string? 🤔  Based on the following code, this is used as a list only.